engine/src/timing.rs

```diff
@@ -0,0 +1,41 @@
+use std::time::Instant;
+
+use cimvr_engine_interface::FrameTime;
+
+pub struct Timing {
```

Comment:
Comment what the timing struct keeps track of please. I know it just keeps track of deltatime and frametime but it would be tedious to check that based off the functions.